checker/src/call_visitor.rs

```diff
@@ -2157,15 +2179,15 @@ impl<'call, 'block, 'analysis, 'compilation, 'tcx, E>
                 }
 
                 // Analyze the tag type's first parameter to obtain a compile-time constant.
-                let tag_mask_abs = self
+                let tag_propagation_set_abs = self
                     .block_visitor
-                    .visit_constant(None, tag_mask_rustc_const);
+                    .visit_constant(None, tag_propagation_set_rustc_const);
                 if let Expression::CompileTimeConstant(ConstantDomain::U128(data)) =
-                    &tag_mask_abs.expression
+                    &tag_propagation_set_abs.expression
                 {
                     let tag_kind = tag_adt_def.did;
-                    let tag_mask = *data;
-                    Some((tag_kind, tag_mask))
+                    let tag_propagation_set = *data;
+                    Some((tag_kind, tag_propagation_set))
                 } else {
                     assume_unreachable!(
```

Comment:
Looking at this again, I am wondering if this is actually unreachable. Can you add a comment to convince that this can never be reached?